src/pages/index.vue

```diff
@@ -113,3 +42,92 @@
     }
   }
 </script>
+
+<template>
+  <div class="h-100 ">
+    <VContainer 
+      class="
+        h-100 
+        d-flex 
+        flex-column 
+        justify-center 
+        align-center
+      "
+    >
+      <VCard
+        width="400"
+      >
+        <VForm
+          v-model="isFormValid"
+          class="pa-4"
+        >
+          <VRow justify="center">
+            <VCol cols="10">
+              <VTextField
+                v-model="username"
+                :rules="[
+                  $options.rules.invalidChars,
+                  $options.rules.required, 
+                  $options.rules.min
+                ]"
+                counter
+                density="comfortable"
+                label="Username"
+                variant="underlined"
+              />
+            </VCol>
+          </VRow>
+      
+          <VRow justify="center">
+            <VCol cols="10 md-7">
```

Comment:
If I'm not mistaken, this shouldn't work correctly.

What you likely wanted to do can be done with:

```suggestion
            <VCol
              cols="10"
              md="7"
            >
```

--> 10 cols by default, 7 cols on >= medium sized devices

(the same applies to the following `VCol` elements).